locale/GoogleEarthWallpaper.pot

```diff
@@ -107,114 +107,90 @@ msgstr ""
 msgid "Select how the image is rendered on desktop"
 msgstr ""
 
-#: extension.js:97
+#: extension.js:95
 msgid "<No refresh scheduled>"
 msgstr ""
 
-#: extension.js:98
+#: extension.js:96
 msgid "Text Location"
 msgstr ""
 
-#: extension.js:99
+#: extension.js:97
 msgid "Geo Location"
 msgstr ""
 
-#: extension.js:100
+#: extension.js:98
 msgid "External Link"
 msgstr ""
 
-#: extension.js:101
+#: extension.js:99
 msgid "Copyright"
 msgstr ""
 
-#: extension.js:102
+#: extension.js:100
 msgid "Set background image now"
 msgstr ""
 
-#: extension.js:103
+#: extension.js:101
 msgid "Set lockscreen image now"
 msgstr ""
 
-#: extension.js:104
+#: extension.js:102
 msgid "Refresh Now"
 msgstr ""
 
-#: extension.js:105
+#: extension.js:103
 msgid "Extension settings"
 msgstr ""
 
-#: extension.js:113
+#: extension.js:111
 msgid "Set lockscreen image"
 msgstr ""
 
-#: extension.js:114
+#: extension.js:112
 msgid "Send notification"
 msgstr ""
 
-#: extension.js:137
+#: extension.js:131
 msgid "On refresh:"
 msgstr ""
 
-#: extension.js:225
+#: extension.js:209
 msgid "Next refresh"
 msgstr ""
 
-#: extension.js:277
+#: extension.js:261
 msgid "View in "
 msgstr ""
 
-#: extension.js:332
+#: extension.js:319
 msgid "Fetching..."
 msgstr ""
 
-#: extension.js:446
+#: extension.js:434
 msgid "No wallpaper available"
 msgstr ""
 
-#: extension.js:447
+#: extension.js:435
 msgid "Something went wrong..."
 msgstr ""
 
-#: prefs.js:23
-msgid "5 m"
-msgstr ""
-
-#: prefs.js:23
-msgid "10 m"
-msgstr ""
-
-#: prefs.js:23
-msgid "30 m"
-msgstr ""
-
-#: prefs.js:23
-msgid "60 m"
-msgstr ""
-
-#: prefs.js:23
-msgid "90 m"
-msgstr ""
-
-#: prefs.js:23
-msgid "daily"
-msgstr ""
-
-#: prefs.js:112
+#: prefs.js:116
```

Comment:
I just realized `.map(_)` is silly and doesn't seem to work with gettext. will fix that.